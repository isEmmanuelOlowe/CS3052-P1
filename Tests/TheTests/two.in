2
0.0 0.1
1.7 99.9
