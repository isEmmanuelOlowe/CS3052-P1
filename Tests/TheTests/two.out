99.8144779
